### PR TITLE
dimensions/YAxis: correct label/title rotation for computing Y-axis width

### PR DESCRIPTION
**Pull request by karna48**
# YAxis width corrected
Using multiple Y-axis (6) there were visible problems with spaces between them.
yaxis[i].labels.rotate and yaxis[i].title.rotate were not properly used when computing the bounding boxes

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by karna48 — COMMENTED** [review]
I have run the tests and all passes.
Visually multiple Y-axis (more than 1 on one side) looks cleaner

File: src/modules/dimensions/YAxis.js [hunk 1]
```diff
@@ -64,13 +64,25 @@ export default class DimYAxis {
         }
 
         let graphics = new Graphics(this.dCtx.ctx)
-        let rect = graphics.getTextRects(val, yaxe.labels.style.fontSize)
+
+        let rotateStr = 'rotate(' + yaxe.labels.rotate + ' 0 0)'
+        let rect = graphics.getTextRects(
+          val,
+          yaxe.labels.style.fontSize,
+          yaxe.title.style.fontFamily,
```
Comment (junedchhipa) [inline src/modules/dimensions/YAxis.js:72]: It should be `yaxe.labels.style.fontFamily`

File: src/modules/dimensions/YAxis.js [hunk 2]
```diff
@@ -64,13 +64,25 @@ export default class DimYAxis {
...
+        )
+
         let arrLabelrect = rect
 
         if (val !== valArr) {
           arrLabelrect = graphics.getTextRects(
             valArr,
-            yaxe.labels.style.fontSize
+            yaxe.labels.style.fontSize,
+            yaxe.title.style.fontFamily,
```
Comment (junedchhipa) [inline src/modules/dimensions/YAxis.js:83]: It should be `yaxe.labels.style.fontFamily`

**Review by junedchhipa — CHANGES_REQUESTED** [review]
Please fix the 2 issues I mentioned.
Rest looks good.

**Review by junedchhipa — APPROVED** [review]

**Pull request merged**